core/src/main/java/org/opensearch/sql/executor/ExecutionEngine.java

```diff
@@ -50,9 +50,23 @@ void execute(PhysicalPlan plan, ExecutionContext context,
    * Data class that encapsulates ExprValue.
    */
   @Data
+  @RequiredArgsConstructor
   class QueryResponse {
     private final Schema schema;
     private final List<ExprValue> results;
+    private final String rawResponse;
```

Comment:
can you add a comment on what this is for?

Reply:
Changed in https://github.com/Bit-Quill/opensearch-project-sql/commit/c7208ea1f35ee9d657fb6898e800edb77e142ab3